agent/engine/docker_task_engine.go

```diff
@@ -942,6 +942,8 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	container.SetLabels(config.Labels)
 	seelog.Infof("Task engine [%s]: created docker container for task: %s -> %s, took %s",
 		task.Arn, container.Name, metadata.DockerID, time.Since(createContainerBegin))
+	container.SetRuntimeId(metadata.DockerID)
```

Comment:
I noticed there is docker id empty check on line 935, do we want to also include this there or we don't care here that docker id is empty?

Reply:
When create the container failed, we leave the container's dockerId as null.